packages/bsky/src/api/app/bsky/unspecced/getTrendingTopics.ts

```diff
@@ -48,6 +48,7 @@ const skeleton = async (input: SkeletonFnInput<Context, Params>) => {
   if (ctx.topicsAgent) {
     const res = await ctx.topicsAgent.app.bsky.unspecced.getTrendingTopics(
       {
+        ...params,
```

Comment:
If I'm reading correctly, this will include some values we don't want to try to pass through such as `headers`.  Can we pass through the specific fields accepted by `unspecced.getTrendingTopics` instead?

Reply:
oh, yea. is this actually how you all usually structure these? (i.e., in my head was just thinking `params` was "request params" and didnt look closely. ill push change for now)

Reply:
I'm personally paranoid and try to be explicit at service boundaries, but I wouldn't be surprised at all if there are exceptions to that written by me in this codebase 😅